cfme/tests/test_replication.py

```diff
@@ -145,9 +148,9 @@ def test_replication_delete_remote_from_global(setup_replication):
         region.replication.get_replication_status(host=remote_app.hostname)
 
 
-@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
 @pytest.mark.tier(1)
-def test_replication_remote_to_global_by_ip_pglogical():
+def test_replication_remote_to_global_by_ip_pglogical(setup_replication, provider):
```

Comment:
Below comment will instantiate `provider` object hence no need to pass here, Pass `OpenStackProvider`.